packages/sdk-demo/src/components/Layout/ThemeSelector.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { useRef, useState } from 'react';
+
+import { ThemeConfig } from '@/types';
+import type { I18n } from '@lingui/core';
+import { t } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import { useRootElements } from '@monite/sdk-react';
+import DarkModeIcon from '@mui/icons-material/DarkMode';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp';
+import LightModeIcon from '@mui/icons-material/LightMode';
+import {
+  Button,
+  Typography,
+  Menu,
+  MenuItem,
+  Divider,
+  FormControlLabel,
+  Switch,
+} from '@mui/material';
+
+const getButtonLabel = (themeIndex: string, i18n: I18n) => {
+  switch (themeIndex) {
+    case 'material':
+      return t(i18n)`Material UI`;
+    case 'monite':
+      return t(i18n)`Monite`;
+    default:
+      return t(i18n)`Material UI`;
+  }
+};
+
+interface ThemeSelectorProps {
+  themeConfig: ThemeConfig;
+  onChange: (themeConfig: ThemeConfig) => void;
+}
+
+export const ThemeSelector = ({
+  themeConfig,
```

Comment:
- I would recommend renaming `themeConfig` to `value`, as it would be conventional to _Select_.
- I would prefer to keep MUI naming for the components, so, _ThemeSelector_ probably may be renamed to _ThemeSelect_
- Just Brilliant, btw 🔷

Reply:
good point, thanks. fixed